### ISSUE TITLE
Unpublished Changes pill disappears after rejecting end enrollment

### ISSUE THREAD

**Issue by jaredlockhart** [mozilla/experimenter#8417]
1) Start with a live rollout
2) Edit the population size
3) Unpublished Changes is displayed
4) Click 'End Enrollment'
5) Reject

Expected behaviour:
Back to DIRTY

Actual behaviour:
Back to IDLE

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3235)

**Comment by jrbenny35** [comment]
Would be nice to have this checked within the integration tests in #8378

**Comment by eliserichards** [comment]
This shouldn't be an issue after #8422

**Comment by eliserichards** [comment]
Closing as invalid since we removed the end enrollment button for live rollouts 👍
